README_new.md

```diff
@@ -0,0 +1,132 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+babylon health
+==================================
+
+At babylon, we firmly believe that *transparency* and *flexibility* are core values that should be present in everything we do, and hiring the best person for a position should reflect that belief.
+
+That's why we in the iOS team have open-sourced our recruitment process, so candidates know exactly what to expect, all the way to the questions we may ask a candidate during a face-to-face interview, and so that the assessment process is fair and square.
```

Comment:
~a candidate~

Reply:
> a face-to-face interview. We are firm believers that this will make the assessment process fairer.

Reply:
Fixed in https://github.com/Babylonpartners/iOS-Interview-Demo/pull/4/commits/a8d7ac920ff3897cc7283ee42210ade21b4fb773.